include_core/omrport.h

```diff
@@ -872,16 +872,16 @@ typedef struct J9ProcessorInfos {
 #define OMRPORT_CTLDATA_VMEM_ADVISE_HUGEPAGE  "VMEM_ADVISE_HUGEPAGE"
 #define OMRPORT_CTLDATA_VMEM_PERFORM_FULL_MEMORY_SEARCH  "VMEM_PERFORM_FULL_SEARCH"
 #define OMRPORT_CTLDATA_VMEM_HUGE_PAGES_MMAP_ENABLED "VMEM_HUGE_PAGES_MMAP_ENABLED"
-#define OMRPORT_CTLDATA_INSTANTON_FLAGS "INSTANTON_FLAGS"
+#define OMRPORT_CTLDATA_CRIUSUPPORT_FLAGS "CRIUSUPPORT_FLAGS"
```

Comment:
If we use `cruiSupport`, then there will be a `_` between `CRIU` and `SUPPORT`: `CRIU_SUPPORT`.
```suggestion
#define OMRPORT_CTLDATA_CRIU_SUPPORT_FLAGS "CRIU_SUPPORT_FLAGS"
```

Reply:
Updated.

@babsingh please have another look.